cobbler/items/distro.py

```diff
@@ -115,10 +115,10 @@ def check_if_valid(self):
         """
         if self.name is None:
             raise CX("name is required")
-        if self.kernel is None:
-            raise CX("Error with distro %s - kernel is required" % self.name)
-        if self.initrd is None:
-            raise CX("Error with distro %s - initrd is required" % self.name)
+        if self.kernel is None and self.remote_boot_kernel is None:
+            raise CX("Error with distro %s - either kernel or remote-boot-kernel is required" % self.name)
```

Comment:
So here and below you say in the message that one of those two need to be specified but in the condition you have written `and`. Is this on purpose and if yes why does that work?

Reply:
I think it is an error if both are `None`. If either is set, this is supposed to be fine.

In addition I think it is an error if both are set. But this is not coded, and may be a separate change.

Slightly related: `cobbler sync` still requires a local kernel+initrd, it will error out. Also this should be addressed in a separate change.